Ports/.port_include.sh

```diff
@@ -505,6 +503,9 @@ do_configure() {
     ensure_build
     if [ "$useconfigure" = "true" ]; then
         echo "Configuring $port..."
+        if "$use_fresh_config_sub"; then
+            ensure_new_config_sub
+        fi
```

Comment:
Lots of ports redefine `configure()`, this won't work either I'm afraid.

Reply:
It may be unclear from looking at diff, but this piece is located in `do_configure()`, not `configure()`, and gets called before `pre_configure()`, `configure()` and `post_configure()`, so this works as inteded and cannot be redefined.

Reply:
Ah, I see. that's fine then :^)